tasks/transform_conformance/tests/babel-plugin-transform-typescript/test/fixtures/use-define-for-class-fields-without-class-properties/output.js

```diff
@@ -0,0 +1,63 @@
+
+let _y, _y2, _y3, _y4, _a, _c, _y5, _a2, _c2;
+
+class Cls {
+  constructor() {
+    this.y = 1;
+    this[_y] = 1;
+  }
+  static {
+    x;
+    _y = y;
+    z;
+  }
+}
+
+class ClsWithConstructor extends Cls {
+  static {
+    x;
+    _y2 = y;
+    z;
+  }
+  constructor() {
+    console.log("constructor");
+    super();
+    this.y = 1;
+    this[_y2] = 1;
+  }
+}
+
+class StaticCls {
```

Comment:
```suggestion
class StaticCls {
  static {
    _y3 = y;
  }
```
I guess this is missing here.

Reply:
Good catch! 

Oxc output is:

```js
class StaticCls {
  static {
    x;
    _y3 = y;
    z;
  }
  static {
    this.y = 1;
  }
  static {
    this[_y3] = 1;
  }
}
```

We treat the computed key needs to be kept or not differently from `TypeScript`, I've noted in https://github.com/oxc-project/oxc/blob/993f6518a174bed3ced3cccc4fd93bdb857fce28/crates/oxc_transformer/src/typescript/class.rs#L112-L116. Keep it as what we do in other places.